libraries/standard/json/cbmc/sources/skipGeneric.c

```diff
@@ -0,0 +1,33 @@
+#include "skipGeneric.h"
+
+/**
+ * See skipGeneric.h for docs
+ *
+ * Advance buffer index beyond some minimum value.
+ */
+static bool_ skipGeneric( const char * buf,
+                          size_t * start,
+                          size_t max,
+                          size_t min )
+{
+    bool_ ret = nondet_bool() ? true : false;
```

Comment:
What are the preconditions on buf and start? We should assert them.

Reply:
added